It's just markdown failing on linked URLs it can't resolve... this is really frustrating, I wish we could override it with a check or something.
unit-test-11 was flaking before due to

> Caused by: org.gradle.internal.resource.transport.http.HttpErrorStatusCodeException: Could not GET 'https://jcenter.bintray.com/com/google/googlejavaformat/google-java-format/1.7/google-java-format-1.7.pom'. Received status code 502 from server: Bad Gateway

--- a/changelog/@unreleased/pr-1338.v2.yml
+++ b/changelog/@unreleased/pr-1338.v2.yml
@@ -1,0 +1,7 @@
+type: fix
+fix:
+  description: Fix task dependencies onto other subprojects for `com.palantir.baseline-exact-dependencies`
+    tasks (`checkUnusedDependencies`, `checkImplicitDependencies`), so they work with
+    gradle 6.4.
+  links:
+  - https://github.com/palantir/gradle-baseline/pull/1338

--- a/gradle-baseline-java/src/main/groovy/com/palantir/baseline/tasks/CheckImplicitDependenciesTask.java
+++ b/gradle-baseline-java/src/main/groovy/com/palantir/baseline/tasks/CheckImplicitDependenciesTask.java
@@ -38,6 +38,7 @@ import org.gradle.api.provider.ListProperty;
 import org.gradle.api.provider.Property;
 import org.gradle.api.provider.Provider;
 import org.gradle.api.provider.SetProperty;
+import org.gradle.api.tasks.Classpath;
 import org.gradle.api.tasks.Input;
 import org.gradle.api.tasks.InputFiles;
 import org.gradle.api.tasks.TaskAction;
@@ -151,7 +152,7 @@ public class CheckImplicitDependenciesTask extends DefaultTask {
         this.dependenciesConfigurations.add(Objects.requireNonNull(dependenciesConfiguration));
     }
 
-    @InputFiles
+    @Classpath
     public final Provider<FileCollection> getSourceClasses() {
         return sourceClasses;
     }

--- a/gradle-baseline-java/src/main/groovy/com/palantir/baseline/tasks/CheckUnusedDependenciesTask.java
+++ b/gradle-baseline-java/src/main/groovy/com/palantir/baseline/tasks/CheckUnusedDependenciesTask.java
@@ -38,6 +38,7 @@ import org.gradle.api.provider.ListProperty;
 import org.gradle.api.provider.Property;
 import org.gradle.api.provider.Provider;
 import org.gradle.api.provider.SetProperty;
+import org.gradle.api.tasks.Classpath;
 import org.gradle.api.tasks.Input;
 import org.gradle.api.tasks.InputFiles;
 import org.gradle.api.tasks.TaskAction;
@@ -206,7 +207,7 @@ public class CheckUnusedDependenciesTask extends DefaultTask {
         this.sourceOnlyConfigurations.add(Objects.requireNonNull(configuration));
     }
 
-    @InputFiles
+    @Classpath
     public final Provider<FileCollection> getSourceClasses() {
         return sourceClasses;
     }

--- a/gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineExactDependenciesTest.groovy
+++ b/gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineExactDependenciesTest.groovy
@@ -19,6 +19,7 @@ package com.palantir.baseline
 import java.nio.file.Files
 import org.gradle.testkit.runner.BuildResult
 import org.gradle.testkit.runner.TaskOutcome
+import spock.lang.Unroll
 
 class BaselineExactDependenciesTest extends AbstractPluginTest {
 
@@ -113,7 +114,8 @@ class BaselineExactDependenciesTest extends AbstractPluginTest {
         result.task(':checkUnusedDependenciesMain').getOutcome() == TaskOutcome.SUCCESS
     }
 
-    def 'checkUnusedDependencies correctly picks up project dependency on java-library'() {
+    @Unroll
+    def '#task correctly picks up project dependency on java-library'() {
         when:
         buildFile << standardBuildFile
         buildFile << """
@@ -139,8 +141,11 @@ class BaselineExactDependenciesTest extends AbstractPluginTest {
         """.stripIndent()
 
         then:
-        def result = with('checkUnusedDependencies', '--stacktrace').build()
+        def result = with(":${task}", '--stacktrace').build()
         assert result.task(':needs-building-first:compileJava').getOutcome() != null
+
+        where:
+        task << ['checkUnusedDependencies', 'checkImplicitDependencies']
     }
 
     def 'checkUnusedDependenciesTest passes if dependency from main source set is not referenced in test'() {

